### ISSUE TITLE
Feature request: Add option to disable centre resizing.

### ISSUE THREAD

**Issue by Bronzdragon** [stefansundin/altdrag#161]
Currently, using altdrag, you can choose to dedicate an action to resizing the window. Which edge gets resized depends on where the cursor is in relation to the window. Moving it on the right side and moving the mouse will resize the right edge, etc. (If a corner is hit, you can resize two edges at once)

It allows you to resize all sides at once, by positioning your cursor near the centre and moving left to right/right to left to scale horizontal, and up and down/down and up to scale vertical. 

I personally am quite lazy with how far I move my cursor, which means I end up resizing the whole window instead of just the edge I want. When that happens, I often have to move the window to a new position. I never use the central resizing on purpose, so an option to disable it would be appreciated. Extending the 'hitboxes' would be appreciated, but is not an absolute requirement.

**TL;DR**: Please add a toggle to disable dragging from the centre.

**Comment by RamonUnch** [comment]
In the unofficial [AltSnap](https://github.com/RamonUnch/AltSnap) fork that I am maintaining, I allow several options for the center resize mode, and one of them is to extend the closest side in to the center, so there is no more this funny *all directions* resize mode.
